    /**
     * Bug 37684  : Unhandled Continue Record Error
     */
    @Test
    public void bug37684() throws Exception {
        HSSFWorkbook wb1 = openSample("37684-1.xls");
        writeOutAndReadBack(wb1).close();
        wb1.close();
        HSSFWorkbook wb2 = openSample("37684-2.xls");
        writeOutAndReadBack(wb2).close();
        wb2.close();
    }

